### PR TITLE
SSH: Deprecated methods are removed.

### PR DESCRIPTION
**Pull request by ismail-yilmaz**
Some deprecated functions were marked for removal in U++ 2021.
This pull requests removes those methods.

### CODE REVIEW

File: uppsrc/Core/SSH/Session.cpp
Comment (klugier) [inline uppsrc/Core/SSH/Session.cpp:209]: ThrowError(-1) is enigmatic. It should ThrowError with information...

Comment (ismail-yilmaz) [reply uppsrc/Core/SSH/Session.cpp:209]: @klugier 

It is a *generic failure* or rejection, separated from the libssh2's own error codes. Since the WhenVerify() hands the operation over to the user. (client), client code *should* deal with it (i.e. it can print a cancellation message or pop up an error dialog.)

Comment (ismail-yilmaz) [reply uppsrc/Core/SSH/Session.cpp:209]: Besides, setting the error code to -1, first checks for errors on libssh2 subsystem. (So it will print an error message if the failure is pertaining to the ssh subsystem.)

**Review by klugier — APPROVED** [review]
Looks good to me, however you should wait for Mirek's approval!

**Pull request merged**
